java/semantickernel-connectors-parent/semantickernel-connectors/src/main/java/com/microsoft/semantickernel/connectors/memory/sqlite/Database.java

```diff
@@ -0,0 +1,290 @@
+// Copyright (c) Microsoft. All rights reserved.
+package com.microsoft.semantickernel.connectors.memory.sqlite;
+
+import reactor.core.publisher.Mono;
+import reactor.core.scheduler.Schedulers;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Database {
+
+    static class DatabaseEntry {
```

Comment:
I think dotnet are wrong to have each connector have it's own DatabaseEntry. There is a DataEntryBase that has the key and timestamp. We might want to create a VectorDataEntry that sits along side it that has embedding and metadata in it and use that here (and in the others).

Reply:
For later.